### PR TITLE
Remove product name from strings

### LINKED ISSUE #991: Fix remaining Lockbox strings
While setting up Lockwise iOS for localization, we found a few remaining occurrences of "Lockbox", some in UI, some in a11y, some in comments.

It'd be nice if we could get a fix for those before we expose strings for real.

You can search https://github.com/mozilla-l10n/lockwiseios-l10n/blob/5d9ab487f805536ebe9d9fd003ce783009a103c0/templates/lockwise-ios.xliff for examples.

https://github.com/mozilla-lockwise/lockwise-ios/blob/e64edee308d0a8f37153f17199f2df1d70cac8f9/CredentialProvider/Storyboard/Base.lproj/CredentialWelcome.storyboard#L46 shows that this is at least in spots still happening.

I invented the "blocks-l10n" label for issues like this.

CC @Delphine @devinreams

### PR DESCRIPTION
**Pull request by joeyg**
Fixes #991

### CODE REVIEW

File: CredentialProvider/Storyboard/Base.lproj/CredentialWelcome.storyboard [hunk 1]
```diff
@@ -41,7 +41,7 @@
...
                                 <rect key="frame" x="163" y="359" width="88" height="88"/>
                             </imageView>
-                            <imageView userInteractionEnabled="NO" contentMode="center" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="lockbox-title" adjustsImageSizeForAccessibilityContentSizeCategory="YES" translatesAutoresizingMaskIntoConstraints="NO" id="ehm-2v-Rbb" userLabel="Lockbox">
+                            <imageView userInteractionEnabled="NO" contentMode="center" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="lockbox-title" adjustsImageSizeForAccessibilityContentSizeCategory="YES" translatesAutoresizingMaskIntoConstraints="NO" id="ehm-2v-Rbb" userLabel="Product Name">
                                 <rect key="frame" x="92" y="459" width="230" height="54"/>
                                 <accessibility key="accessibilityConfiguration" label="Firefox Lockbox">
```
Comment (Pike) [inline CredentialProvider/Storyboard/Base.lproj/CredentialWelcome.storyboard]: This one is what's showing up in the export, I think. And this wants to be Lockwise, not Product Name (not sure what the change above does).

**Comment by devinreams** [comment]
The resulting output of the export scripts is shown here:
https://github.com/mozilla-l10n/lockwiseios-l10n/pull/4

Any other changes need up here or downstream?

**Comment by Pike** [comment]
I've left some a comment on the import PR at https://github.com/mozilla-l10n/lockwiseios-l10n/pull/4#pullrequestreview-240462122.

In particular, there's still some "Lockbox" literal being imported back, which we'd like to get fixed still.

**Comment by devinreams** [comment]
Note to future self: where is the CFBundleName coming from? I can't tell how those strings get interpolated..

**Comment by Pike** [comment]
> Note to future self: where is the CFBundleName coming from? I can't tell how those strings get interpolated..

https://github.com/mozilla-lockwise/lockwise-ios/blob/e21883649b616cf02652a832064d8c031a5b64bf/lockbox-ios/Common/Resources/Info.plist#L15-L16 I think

**Comment by devinreams** [comment]
Results of latest export based on this branch is up here (no more Lockbox!):
https://github.com/mozilla-l10n/lockwiseios-l10n/pull/5

**Comment by devinreams** [comment]
Oh I got aggressive and need to not rename the app quite so much.

**Comment by devinreams** [comment]
>I can't pin them down to anything in this patch, though, did something else change?

Thanks @Pike.

I think it would have to be when we removed and then re-added (to clean out duplicate file inclusions) in Xcode here: #1011

**Pull request merged**